charms/README.md

```diff
@@ -0,0 +1,12 @@
+# how to deploy
+
+```bash
+
+charm build ./mysql -o ./
```

Comment:
My preference is to use a builds directory outside the charm's directory.
This avoids polluting each charm dir and allows easier deployment because you can just sit in a builds dir in a shell and deploy ./mysql and deploy ./gitlab from the one shell

Reply:
agreed, do u think we can put builds to `-o ~/.local/charms/`, so it will be `-o ~/.local/charms/builds/*`